website/content/partials/datasource/http/Config-not-required.mdx

```diff
@@ -0,0 +1,5 @@
+<!-- Code generated from the comments of the Config struct in datasource/http/data.go; DO NOT EDIT MANUALLY -->
```

Comment:
You can use these generated partials in the docs. See the other datasources documentation as an example.

Reply:
Updated PR with docs